package.json

```diff
@@ -11,12 +11,15 @@
     "build": "tsc",
     "start": "tsx index.ts",
     "dev": "tsx watch index.ts",
-    "test": "echo \"Error: no test specified\" && exit 1"
+    "test": "echo \"Error: no test specified\" && exit 1",
+    "install:challenges": "tsx install.ts"
   },
   "dependencies": {
+    "axios": "^1.7.2",
```

Comment:
I used `axios` because I saw Carlos using it for the `speedrun-grader`. The main benefits for this repo relate to error handling and automatic JSON data transformation

Axios treats any status code outside the 200 range as an error and will reject the promise in such cases, which simplifies error handling logic. On the other hand, Fetch will only reject on network errors and requires you to manually throw errors for status codes that indicate an error.

It was more convenient and faster to use axios so i could more closely follow Carlos example, but I won't contest if you decide fetch is better choice.

Also if you do feel strongly about minimizing dependencies perhaps `express-validator` should also be removed? Maybe that should be a separate issue to refactor?

Reply:
Let's keep axios and express-validator and if they become a problem (doubt they will) then we will refactor.